config/index.js

```diff
@@ -271,6 +271,25 @@ var conf = convict({
       default: ''
     }
   },
+  verificationReminder: {
+    enabled: {
+      doc: 'Is verification reminder enabled',
+      format: Boolean,
+      env: 'VERIFICATION_REMINDER_ENABLED',
+      default: false
+    },
+    queueUrl: {
+      doc: 'The verification reminder queue URL to use (should include https://sqs.<region>.amazonaws.com/<account-id>/<queue-name>)',
+      format: String,
+      env: 'VERIFICATION_REMINDER_QUEUE_URL',
```

Comment:
First to confirm, I assume it is a `Yes` to 

> are queue permissions guarded by QueuePolicy?

Otherwise: 
> What do you mean by locally, non-AWS like local-dev?

For local dev we were planning to have a local (non-AWS) queue to manage these (for self-hosters and non-AWS deployments as well). 

However I wanted to test real SQS locally, would the `aws-sdk` module look for my AWS creds via `process.env`?